### DISCUSSION TITLE
Ability to share tables and explorations

### DISCUSSION THREAD

**Discussion by kgodey** [mathesar-foundation/mathesar#2266] [Feature Ideas]
Mathesar should support the ability to share tables and explorations (or even entire databases and schemas) publicly. 

---
*If you'd like us to build this feature, please comment below with more details about how you plan to use this functionality.*

**Comment by pavish** [comment]
As of [Release 0.1.3](https://github.com/centerofci/mathesar/releases/tag/0.1.3), Mathesar supports sharing read only views of tables and explorations via public URLs.
